december.js

```diff
@@ -5290,6 +5291,7 @@ class LoopyEffect {
     root.style.setProperty('--loopy-random-5', Math.random());
 
     LoopyEffect.setupContainer();
+    LoopyEffect.state.container.classList.add('c-loopy-container');
```

Comment:
![Image](https://github.com/user-attachments/assets/bc101674-887b-4767-b516-8a1c8a1ccb7c)

These changes should fix the misplaced user tooltips and z-layer issues after loopy. See image for reference.

Reply:
oh yo sick nice